f8a_worker/storages/postgres.py

```diff
@@ -286,9 +289,13 @@ def validate_utf_json(result):
     :param result: dict, Dictionary having details of task
     return: a dictionary received as input having non UTF characters removed if present
     """
-    if 'details' in result and isinstance(result['details'], list) \
-            and len(result['details']) > 0 and 'description' in result['details'][0]:
-        result['details'][0]['description'] = ''.join(
-            x for x in result['details'][0]['description']
-            if x in string.printable)
-    return result
+    try:
+        if result is not None and 'details' in result and isinstance(result['details'], list) \
+                and len(result['details']) > 0 and 'description' in result['details'][0]:
+            result['details'][0]['description'] = ''.join(
+                x for x in result['details'][0]['description']
+                if x in string.printable)
+        return result
+    except Exception as e:
```

Comment:
Also I'm little hesitant to use base Exception class to handle it. It would mask the real issue which may need our attention. Can you use specific types of Exceptions? (possible exception classes could be KeyError, ValueError)

Reply:
Okay @arajkumar I will keep multiple catch blocks for KeyError and ValueError but just to be on safe side I will also keep base Exception as well.

Reply:
@jparsai I don't recommend to use base Exception class unless there is a valid reason for that to have.

Reply:
You can do the following instead of multiple blocks,
```
except (KeyError, ValueError) as e:
  ...
```